paddle/fluid/operators/log_loss_op.cc

```diff
@@ -23,25 +23,36 @@ class LogLossOp : public framework::OperatorWithKernel {
   using framework::OperatorWithKernel::OperatorWithKernel;
 
   void InferShape(framework::InferShapeContext* ctx) const override {
-    PADDLE_ENFORCE(ctx->HasInput("Predicted"),
-                   "Input(Predicted) must be initialized.");
-    PADDLE_ENFORCE(ctx->HasInput("Labels"),
-                   "Input(Labels) must be initialized.");
+    OP_INOUT_CHECK(ctx->HasInput("Predicted"), "Input", "Predicted", "LogLoss");
+    OP_INOUT_CHECK(ctx->HasInput("Labels"), "Input", "Labels", "LogLoss");
 
     auto pred_dims = ctx->GetInputDim("Predicted");
     auto label_dims = ctx->GetInputDim("Labels");
 
     if (ctx->IsRuntime() || (framework::product(pred_dims) > 0 &&
                              framework::product(label_dims) > 0)) {
-      PADDLE_ENFORCE_EQ(pred_dims, label_dims);
+      PADDLE_ENFORCE_EQ(
+          pred_dims, label_dims,
+          platform::errors::InvalidArgument(
+              "The rank of Input(Predicted) must be equal to the"
+              "rank of Input(Labels), but received rank of Input(Predicted)"
+              "is [%s], received rank of Input(Labels) is [%s].",
+              pred_dims, label_dims));
     }
     PADDLE_ENFORCE_EQ(pred_dims.size(), 2,
-                      "The rank of Input(Predicted) must be 2 and the shape is "
-                      "[batch_size, 1].");
+                      platform::errors::InvalidArgument(
+                          "The rank of Input(Predicted) must be 2,"
+                          "But received rank of Input(Predicted)"
```

Comment:
`rank` -> `dimension`? refer https://github.com/PaddlePaddle/Paddle/pull/20099, other places also need be changed